ts/components/ShortcutGuide.tsx

```diff
@@ -155,6 +156,7 @@ function getNavigationShortcuts(i18n: LocalizerType): Array<ShortcutType> {
       id: 'Keyboard--open-sticker-chooser',
       description: i18n('icu:Keyboard--open-sticker-chooser'),
       keys: [['commandOrCtrl', 'shift', 'G']],
+      note: 'Opens the sticker picker in the current conversation',
```

Comment:
What is the point of `note` here? Isn't this documentation that should be done in the i18n itself?